<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addForm</class>
 <widget class="QWidget" name="addForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>671</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить данные</string>
  </property>
  <widget class="QPushButton" name="addRoadButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>240</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить участок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addCrossroadButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить перекрёсток</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="latitudeEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>50.25958</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="longitudeEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>127.54389</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Долгота:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>270</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Широта:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Ремонтируемые участки&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>-10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Объездные участки</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>-10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Группы объезда</string>
   </property>
  </widget>
  <widget class="QLabel" name="imageMap">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>649</width>
     <height>449</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="addBadRoadButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавть участок</string>
   </property>
  </widget>
  <widget class="QListWidget" name="badRoadList">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>21</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="roadList">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="crossroadList">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>171</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="coor">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>240</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;
&lt;body&gt;
  &lt;style&gt;
   p {
    margin-top: 0.1em; /* Отступ сверху */
    margin-bottom: 0.1em; /* Отступ снизу */
   }
  &lt;/style&gt;
&lt;p&gt; &lt;/p&gt;
&lt;p&gt; &lt;/p&gt;
&lt;/body&gt;
&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>242</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>171</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Карта и нагрузка</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string notr="true">Карта</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Нагрузка</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="delBadButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить участок</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delCrossroadButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>80</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить группу</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delRoadButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить улицу</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
